traffic_portal/app/src/common/modules/form/deliveryService/edit/FormEditDeliveryServiceController.js

```diff
@@ -299,7 +299,10 @@ var FormEditDeliveryServiceController = function(deliveryService, origin, topolo
 				deliveryServiceService.deleteDeliveryService(deliveryService)
 					.then(
 						function() {
-							messageModel.setMessages([ { level: 'success', text: 'Delivery service [ ' + deliveryService.xmlId + ' ] deleted' } ], true);
+							messageModel.setMessages([
```

Comment:
why? TP can just show the message(s) sent from the API. This approach has essentially hard-coded the api message.

Reply:
two main reasons

1. That's what it already was doing, so this PR merely isn't a fix for it - it's not introducing the unwanted behavior.
2. This hard-codes the response, because all Traffic Ops returns is "ds was deleted". The custom Alert puts the XMLID in the message. Fixing Traffic Ops to include that information is involved, because Delivery Services use the CRUDer, so it's difficult to change nearly anything about the endpoint without retooling the whole API. Which is out of scope for this change.

Reply:
Exactly what Brennan said. He had the same question and I explained why it is difficult to change things on TP since cruder on TO gives a generic message, hence developers of TP set a different message than results.alerts.

Reply:
but i see

`				alerts := tc.CreateAlerts(tc.SuccessLevel, message)
				alerts.AddNewAlert(tc.InfoLevel, "Perform a CDN snapshot then queue updates on all servers in the cdn for the changes to take effect.")
`

why not just show those. i think it's fine that the xmlId is lost in the UI.

Reply:
I'm not sure either. It should be changed, but that can be done in a separate PR.

Reply:
@ocket8888 @mitchell852 I am creating another to remove hardcoding and updating test to use the updated validationMessage.